### DISCUSSION TITLE
刚搬到新艾利都

### DISCUSSION THREAD

**Discussion by Lanmuqiao** [share121/inter-knot#680] [其他]
![image](https://github.com/user-attachments/assets/002e0bbc-5464-499c-b397-7f8ca75a3a13)
刚搬到新艾利都，花了100000000丁尼买了个房子，不知道大家有没有推荐的景点

**Comment by Yizakl** [comment]
零号空洞

**Reply by Lanmuqiao** [reply]
谢谢你啊，热心的本地人，现在正在和尼尼微喝茶

**Comment by lllliufeng** [comment]
旧都广场

**Comment by Yizakl** [comment]
零号空洞不仅无需门票，且风景优美，必扬的（值得表扬的）

**Comment by mig252** [comment]
那必须的是六分街，民风淳朴

**Comment by Mu-7593** [comment]
楼主以前是生活在外环吗？想问问楼主是怎么赚到那么多钱买房子的

**Reply by Lanmuqiao** [reply]
偷偷告诉你，我是干绳斤的

**Comment by GanlinSama** [comment]
wok，这是有钱人啊
